Core/Public/HyperionManager.m

```diff
@@ -202,7 +202,14 @@ -(void)setActivationGesture:(HYPActivationGestureOptions)gesture {
         if ([pluginClass conformsToProtocol:@protocol(HYPPlugin)])
         {
             id <HYPPluginModule> module = [pluginClass createPluginModule:pluginExtension];
-            [mutablePluginModules addObject:module];
+            if (module)
+            {
+                [mutablePluginModules addObject:module];
+            }
+            else
+            {
+                NSLog(@"%@ createPluginModule returned a nil plugin module ", NSStringFromClass(pluginClass));
```

Comment:
This looks great! Can you please remove the extra space after module?

Reply:
Good catch